testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -473,6 +473,35 @@ testModule('option', moduleConfig, () => {
         assert.strictEqual(onResizeFired.callCount, 1, 'onResize fired');
         assert.strictEqual(onResizeEndFired.callCount, 1, 'onResizeEnd fired');
     });
+
+    test('wrapperAttr defaultOption', function(assert) {
+        Overlay.defaultOptions({
+            options: {
+                wrapperAttr: { id: 'deafultId' }
+            }
+        });
+
+        const overlay = new Overlay('#overlay');
+        const $content = $(overlay.content());
+        const $wrapper = $content.parent();
+        assert.strictEqual($wrapper.attr('id'), 'deafultId', 'defaultOption for wrapperAttr works properly');
+    });
+
+    test('wrapperAttr', function(assert) {
+        const $overlay = $('#overlay').dxOverlay({
+            wrapperAttr: { class: 'class', id: 'id', anotherAttr: 'someValue' }
```

Comment:
Test should be simple. Do we really need all this attributes or the single is enough for functionality testing?